src/libopensc/card-oberthur.c

```diff
@@ -1125,9 +1125,9 @@ auth_compute_signature(struct sc_card *card, const unsigned char *in, size_t ile
 	apdu.datalen = ilen;
 	apdu.data = in;
 	apdu.lc = ilen;
-	apdu.le = olen > 256 ? 256 : olen;
+	apdu.le = MIN(olen, 256);
```

Comment:
I would use the `SC_MAX_APDU_RESP_SIZE` here too, instead of the magic number. And likely too for the `resp` buffer definition above as it has the same issue as the `read_record()` below.